packages/dd-trace/test/telemetry/logs/log-collector.spec.js

```diff
@@ -63,7 +63,7 @@ describe('telemetry log collector', () => {
       })).to.be.true
     })
 
-    it('should not include original message if first frame is not a dd frame', () => {
+    it('should include original message if first frame is not a dd frame', () => {
```

Comment:
Why is it okay that this got reversed?

Reply:
we were using the Error message as telemetry log message, and we were replacing it by 'omitted' when first frame is not a dd frame.
But now as we are making mandatory to provide a message there is no need to replace it.